src/main/java/Excavator.java

```diff
@@ -0,0 +1,9 @@
+class Excavator extends Machine {
+    public void doWork() {
+        System.out.println("Excavator started working.");
```

Comment:
The message in `doWork` method should be more informative and indicate that it is an `Excavator` that is working. Consider including the class name dynamically using `this.getClass().getSimpleName()`.